statsmodels/base/model.py

```diff
@@ -804,11 +804,12 @@ def predict(self, exog=None, transform=True, *args, **kwargs):
         if transform and hasattr(self.model, 'formula') and (exog is not None):
             from patsy import dmatrix
             if isinstance(exog, pd.Series):
-                exog = pd.DataFrame(exog)
+                exog = pd.DataFrame(exog).T
```

Comment:
I'm guessing this caused the Travis fail... 

Maybe count the records or the number of `params`? If `params` > 1, then do the transpose?

Reply:
That's what we do when we don't go through the patsy transform path.
The problem is that if there is a categorical variable or a polynomial expression, then one exog-data variable results in several `exog` columns and several params.